package.json

```diff
@@ -8,14 +8,14 @@
     "create-local": "graph create $THE_GRAPH_GITHUB_USER/$THE_GRAPH_SUBGRAPH_NAME --node http://127.0.0.1:8020",
     "build": "graph build",
     "deploy-local": "graph deploy $THE_GRAPH_GITHUB_USER/$THE_GRAPH_SUBGRAPH_NAME --debug --ipfs http://localhost:5001 --node http://127.0.0.1:8020",
-    "deploy": "graph deploy $THE_GRAPH_GITHUB_USER/$THE_GRAPH_SUBGRAPH_NAME --ipfs https://api.thegraph.com/ipfs/ --node https://api.thegraph.com/deploy/ --debug",
+    "deploy": "graph deploy ianlapham/Uniswap2 --ipfs https://api.thegraph.com/ipfs/ --node https://api.thegraph.com/deploy/ --debug",
```

Comment:
this should be changed back to the generic version no?

Reply:
Yep all this should be generic

Reply:
Looks like we want uniswap/uniswap if this will be merged into master